content/support-matrix/delfin/dell-emc/vmax/_index.md

```diff
@@ -0,0 +1,448 @@
+         </tr>
+     </tbody>
+ </table>
+ 
+ ### Performance registration information
+ 
+ <table>
+     <thead>
+         <tr>
+             <th>Resource</th>
+             <th>Metric</th>
+             <th>Additional information</th>
+         </tr>
+     </thead>
+     <tbody>
+         <tr>
+             <td rowspan=3></td>
+             <td></td>
+             <td></td>
+         </tr>
+         <tr>
+             <td></td>
+             <td></td>
+         </tr>
+         <tr>
+             <td></td>
+             <td></td>
+         </tr>
+         <tr>
+             <td></td>
+             <td></td>
+         </tr>
+     </tbody>
+ </table>
+ 
+ ### Resource data matrix
+ 
+ <table>
+     <thead>
+         <tr>
+             <th>Resource</th>
+             <th>Attribute</th>
+             <th>Additional information</th>
+         </tr>
+     </thead>
+     <tbody>
+         <tr>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               <tr>
+             <td rowspan=9>Pool</td>
+             <td>id</td>
+             <td></td>
+         </tr>
+         <tr>
+             <td>name</td>
+             <td></td>
+         </tr>
+         <tr>
+             <td>storage_id</td>
+             <td></td>
+         </tr>
+         <tr>
+            <td>native_storage_pool_id</td>
```

Comment:
Give user readable filed name , like native_storage_pool_id can be named as 'storage pool id' follow this for all

Reply:
done